tests/MongoDB.Bson.Tests/Jira/CSharp1559Tests.cs

```diff
@@ -14,6 +14,7 @@
 */
```

Comment:
Maybe it makes sense to move these tests to the new file without Jira number in the name. Thoughts?